jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/assignment/LoadBalanceAssignmentStrategyTest.java

```diff
@@ -0,0 +1,252 @@
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+import org.drools.core.rule.constraint.ConditionAnalyzer.ThisInvocation;
+import org.jbpm.services.task.HumanTaskServiceFactory;
+import org.jbpm.services.task.assignment.impl.strategy.LoadBalanceAssignmentStrategy;
+import org.jbpm.services.task.assignment.impl.strategy.RoundRobinAssignmentStrategy;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.kie.api.task.model.Status;
+import org.kie.api.task.model.Task;
+import org.kie.internal.task.api.InternalTaskService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ListMultimap;
+
+import bitronix.tm.resource.jdbc.PoolingDataSource;
+
+public class LoadBalanceAssignmentStrategyTest extends AbstractAssignmentTests {
+    private PoolingDataSource pds;
+    private EntityManagerFactory emf;
+    private ListMultimap<String, Task> tasks;
+    private static final Logger logger = LoggerFactory.getLogger(LoadBalanceAssignmentStrategyTest.class);
+    private static final String BASE_TASK_INFO = "with (new Task()) { priority = 55, taskData = (with (new TaskData()) { } ), ";
+    private static final String MULTI_ACTOR_ASSIGNMENTS = ""
+    		+ "peopleAssignments = (with (new PeopleAssignments()) { potentialOwners = [new User('Bobba Fet'), new User('Darth Vader'), new User('Luke Cage')],"
+            + " businessAdministrators = [new User('Administrator')], } ),";
+    private static final String MULTI_ACTOR_WITH_GROUP_ASSIGNMENTS = ""
+    		+ "peopleAssignments = (with (new PeopleAssignments()) { potentialOwners = [new User('Bobba Fet'), new Group('Crusaders'), new User('Luke Cage')],"
+    		+ " businessAdministrators = [new User('Administrator')], } ),";
+    private static final String ADD_ACTOR_ASSIGNMENTS = ""
+    		+ "peopleAssignments = (with (new PeopleAssignments()) { potentialOwners = [new User('Bobba Fet'), new User('Darth Vader'), new User('Luke Cage'), new User('Tony Stark')],"
+    		+ " businessAdministrators = [new User('Administrator')], } ),";
+    private static final String REMOVE_ACTOR_ASSIGNMENTS = ""
+    		+ "peopleAssignments = (with (new PeopleAssignments()) { potentialOwners = [new User('Bobba Fet'), new User('Luke Cage'), new User('Tony Stark')],"
+    		+ " businessAdministrators = [new User('Administrator')], } ),";
+    private static final String DARTH_VADER = "Darth Vader";
+    private static final String BOBBA_FET = "Bobba Fet";
+    private static final String LUKE_CAGE = "Luke Cage";
+    private static final String TONY_STARK = "Tony Stark";
+
+
+	@Before
+	public void setUp() throws Exception {
+        System.setProperty("org.jbpm.task.assignment.enabled", "true");
+        System.setProperty("org.jbpm.task.assignment.strategy", "LoadBalance");
```

Comment:
I think setting this system property is not necessary since the strategy is later set using `AssignmentServiceProvider` and so the system property is ignored.